frontend/.storybook/preview-head.html

```diff
@@ -0,0 +1 @@
+<link href="/css/main.css" rel="stylesheet" type="text/css" />
```

Comment:
I added this because

1) It does not interfere when hot reloading (when it's working); and
2) It prevents the components from not having any CSS at all in case of failure (sometimes when running storybook from scratch it would pick up the `main.css` styles at all if it failed previously to hot reloaded… must be some internal cache maybe?)